src/custom/components/Popups/ListUpdatePopup.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
```

Comment:
for some files, when we override we create the dir, the index file and the mod file. 
Here we didn't do the directory. Im fine with this too, is ok to decide on a per case basis or should we create the dir to really group the overrided component and it's mod

Reply:
:+1: my thoughts here as well